plugins/flutter_aepedge/README.md

```diff
@@ -21,8 +21,9 @@ Install instructions for this package can be found [here](https://pub.dev/packag
 
 Run:
 
-```bash
-flutter test
+```
+$ cd plugins/flutter_aepedge/
```

Comment:
this readme seems to be under the plugins/flutter_aepedge space, so I think this line is unnecessary.
this comment applies for the other extensions too